pkgs/development/libraries/gtkspell/3.nix

```diff
@@ -1,4 +1,4 @@
-{stdenv, fetchurl, gtk3, aspell, pkgconfig, enchant, isocodes, intltool, gobjectIntrospection}:
+{stdenv, fetchurl, gtk3, aspell, pkgconfig, enchant, isocodes, intltool, gobjectIntrospection, valaSupport ? true, vala_0_40}:
```

Comment:
Let's just build it always with vala support, it should be pretty small and go to dev output anyway.

Reply:
@jtojnar Did the thing